src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -223,17 +223,19 @@ const GroupCreationModal: React.FC<GroupCreationModalProps> = ({
             receipt_mode: enableReadReceipts ? RECEIPT_MODE.ON : RECEIPT_MODE.OFF,
           },
         );
-        setIsShown(false);
 
         if (isKeyboardEvent(event)) {
           createNavigateKeyboard(generateConversationUrl(conversation.qualifiedId), true)(event);
         } else {
           createNavigate(generateConversationUrl(conversation.qualifiedId))(event);
         }
       } catch (error) {
+        amplify.publish(WebAppEvents.CONVERSATION.SHOW, undefined, {});
```

Comment:
I've removed this here https://github.com/wireapp/wire-webapp/pull/15183/files#diff-374d17b2a9898c44a7be4d7d1e40a329f71a2d6b53c865e052296e2e4988646aL219, but it's needed in case of error (it will keep old url and navigate to connections request)